lib/checks/forms/fieldset.js

```diff
@@ -50,28 +50,62 @@ function checkARIAGroup(group, name) {
 	return true;
 }
 
+function checkTable (group, nodes) {
```

Comment:
We could check if they share a table heading? That's what I was originally thinking of doing.